package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "asset-graph-webpack-plugin",
-  "version": "0.1.3",
+  "version": "0.1.4",
```

Comment:
Please revert this change

This will be done automatically via `npm version` command